### PR TITLE
BaSP-MR-115: Redux Activity Screen. 

### PR DESCRIPTION
**Pull request by octavi01**
I worked on creating global states using Redux for the following methods: get, pull, post, getbyid, and delete.

### CODE REVIEW

File: src/redux/activities/reducer.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import {
+  DELETE_ACTIVITY_ERROR,
+  DELETE_ACTIVITY_PENDING,
+  DELETE_ACTIVITY_SUCCESS,
+  GET_ACTIVITIES_ERROR,
+  GET_ACTIVITIES_PENDING,
+  GET_ACTIVITIES_SUCCESS,
+  POST_ACTIVITY_ERROR,
+  POST_ACTIVITY_PENDING,
+  POST_ACTIVITY_SUCCESS,
```
Comment (leo-balonchard) [inline src/redux/activities/reducer.js:10]: please use ADD

File: src/redux/activities/reducer.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import {
+  DELETE_ACTIVITY_ERROR,
+  DELETE_ACTIVITY_PENDING,
+  DELETE_ACTIVITY_SUCCESS,
```
Comment (leo-balonchard) [inline src/redux/activities/reducer.js:4]: please use DEL

File: src/redux/store.js [hunk 3]
```diff
@@ -2,9 +2,11 @@ import { combineReducers, legacy_createStore as createStore, applyMiddleware } f
 import { composeWithDevTools } from 'redux-devtools-extension';
 import thunk from 'redux-thunk';
 import { adminsReducer } from './admins/reducer';
+import { activitiesReducer } from './activities/reducer';
 
 const rootReducer = combineReducers({
-  admins: adminsReducer
+  admins: adminsReducer,
+  activity: activitiesReducer
```
Comment (leo-balonchard) [inline src/redux/store.js:9]: Please use activities

File: src/redux/activities/reducer.js (hunk 1 above)
Comment (octavi01) [reply src/redux/activities/reducer.js]: ready

File: src/redux/activities/reducer.js (hunk 2 above)
Comment (octavi01) [reply src/redux/activities/reducer.js]: ready

File: src/redux/store.js (hunk 3 above)
Comment (octavi01) [reply src/redux/store.js]: ready

**Comment by CavalloFede** [comment]
If you try to add an activity and enter an input with a mistake you are taked back to the list of activities and you should be left on the creations screen, same if you try to edit a class and have a mistake

**Review by nicosimo8 — APPROVED** [review]
Gud job!

**Review by NicolasKler — APPROVED** [review]
Looks good to me

**Review by leo-balonchard — APPROVED** [review]
Ready for merge!

**Pull request merged**